src/libs/actions/Policy.js

```diff
@@ -737,6 +745,22 @@ function generatePolicyID() {
     return _.times(16, () => Math.floor(Math.random() * 16).toString(16)).join('').toUpperCase();
 }
 
+function openWorkspaceMembersPage(policyID, clientPolicyMembers) {
+    if (!policyID) {
```

Comment:
Kinda weird, why do we have this code?

Reply:
The policyID was undefined when viewing the page for a workspace that doesn't exist or the user has no permission. However, I changed this to use the policy ID in the route which is always set so this is no longer needed